examples/ethereum-tracker/src/state.rs

```diff
@@ -5,7 +5,7 @@ use ethereum_tracker::U256Cont;
 use linera_sdk::views::{linera_views, MapView, RegisterView, RootView, ViewStorageContext};
 
 /// The application state.
-#[derive(RootView, async_graphql::SimpleObject)]
+#[derive(RootView)]
```

Comment:
Alternatively, we could use `ComplexObject` and remove the manual implementations of the four getters.